Analyser/AmplFilter/src/com/Main.java

```diff
@@ -0,0 +1,87 @@
+package com.roboy;
+
+import java.io.*;
+import java.math.BigInteger;
+
+public class Main
+{
+    public static void main(String[] args)
```

Comment:
why is it only main function? 
please create separate functions with sensible names according to their functionality and then call it from main